src/Microsoft.Health.Fhir.Liquid.Converter.UnitTests/TestData/TestTemplates/ValidValidateTemplates/Schemas/TestSubPropertySchema.schema.json

```diff
@@ -1,4 +1,5 @@
 {
+    "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
Is this field required?

Reply:
This will be auto-added when convert the **JsonSchema** instant to json string, see [ToJson](https://github.com/RicoSuter/NJsonSchema/blob/25960a0553b88c7e61e229e93c9c6291156b5d04/src/NJsonSchema/JsonSchema.cs#L910)